notebooks/Load_Data_and_EDA.ipynb

```diff
@@ -34,18 +50,99 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "921597ef-c12e-4c4c-b8bf-b1eb20e90814",
+   "metadata": {},
+   "source": [
+    "## Methods"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a0920cdf-10c1-4151-bbf3-a689486257dd",
    "metadata": {},
    "source": [
-    "### Load Data"
+    "### Data"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "012e62cb-71c7-4a4b-8913-d92ac12ce0df",
+   "id": "549db552-150b-4744-ba90-497604b5b601",
    "metadata": {},
    "source": [
-    "Data Source: https://archive.ics.uci.edu/dataset/848/secondary+mushroom+dataset"
+    "The dataset used in this project is the Secondary Mushroom Dataset created by Wagner, D., Heider, D., & Hattab, G. from UCI Machine Learning Repository. This dataset contains 61069 hypothetical mushrooms with caps based on 173 species (353 mushrooms per species). Each mushroom is identified as definitely edible, definitely poisonous, or of unknown edibility and not recommended (the latter class was combined with the poisonous class)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "809c0908-7030-437e-bb4c-56bdf0066119",
+   "metadata": {},
+   "source": [
+    "### Analysis"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "614cbeca-8401-49b3-8eed-cce9dc41d292",
+   "metadata": {},
+   "source": [
+    "The mushroom dataset is balanced with 56% of poisonous mushroom and 44% of edible mushroom. All variables were standardized and vairables with more than 15% missing values are dropped, because imputing a variable that has a significant proportion of missing data might introduce too much noise or bias, making it unreliable. Data was splitted with 80% being partitioned into the training set and 20% being partitioned into the test set. Three classification models including Support Vector Classifier (SVC), K-Nearest Neighbors (KNN), and Logistic Regression are used to predict whether a mushroom is edible or poisonous. The fine tuned Support Vector Classifier has the best overall performance. The hyperparameter was chosen using 5-fold cross validation with F2 score as the classification metric. Beta was chosen to be set to 2 for the F2 score to increase the weight on recall during fitting because redicting a mushroom to be edible when it is in fact poisonous could have severe health consequences. Therefore the goal is to prioritize the minimization of false negative. The Python programming language (Van Rossum and Drake 2009) and the following Python packages were used to perform the analysis: Matplotlib (Hunter, 2007), Pandas (McKinney, 2010), Scikit-learn (Pedregosa et al., 2011), NumPy (Harris et al., 2020), SciPy (Virtanen et al., 2020), UCIMLRepo."
```

Comment:
"**_vairables_**" → "**_variables_**."
"**_redicting_**" → "**_predicting_**."
"minimization of false **_negative_**" → "minimization of false **_negatives_**."